Backend/mentii/class_ctrl.py

```diff
@@ -197,13 +197,16 @@ def getPublicClassList(dynamodb, email=None):
     response.addToPayload('classes', classes)
   return response
 
-def removeStudent(dynamoDBInstance, jsonData, userRole=None):
-  response = ControllerResponse()
+def removeStudent(dynamoDBInstance, jsonData, response=None, userRole=None):
+  currentUserEmail = None
+  if response is None:
+    response = ControllerResponse()
   email = jsonData.get('email')
   classCode = jsonData.get('classCode')
   if g:
     userRole = g.authenticatedUser['userRole']
-  if userRole != 'teacher' and userRole != 'admin':
+    currentUserEmail = g.authenticatedUser['email']
+  if (userRole != 'teacher' and userRole != 'admin') and (currentUserEmail != email):
```

Comment:
uh.. I wouldn't do a comment. But I would change it to match your sentence aka !(userRole == 'teacher' or userRole == 'admin' or currentUserEmail == email) if that makes you sleep better at night?